### PR TITLE
fix: StatefulHA changes added

### PR DESCRIPTION
**Pull request by BRV158**
Missing StatefulHA operator code snippets added

### CODE REVIEW

**Review by bourgeoisor — APPROVED** [review]
LGTM, let me know when this is ready to be merged

**Comment by bourgeoisor** [comment]
@BRV158 gentle ping for the new year

**Comment by bourgeoisor** [comment]
Could you make sure to double check with https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/CONTRIBUTING.md#samples-requirements ?

In particular:
- A README (look at the example linked in the contributing guide)
- A GitHub Action (look at the example linked in the contributing guide) to validate the TF scripts

If the tutorial is going to embed some of this code, then the code to be embed needs to be surrounded by metadata tags so we can embed properly (example: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/quickstarts/languages/deployment.yaml#L16)

Let me know if anything is unclear!

**Comment by BRV158** [comment]
> Could you make sure to double check with https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/CONTRIBUTING.md#samples-requirements ?
> 
> In particular:
> 
> * A README (look at the example linked in the contributing guide)
> * A GitHub Action (look at the example linked in the contributing guide) to validate the TF scripts
> 
> If the tutorial is going to embed some of this code, then the code to be embed needs to be surrounded by metadata tags so we can embed properly (example: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/quickstarts/languages/deployment.yaml#L16)
> 
> Let me know if anything is unclear!

@bourgeoisor - did changes regarding the comments - just a few things to notice at this point 1.- link in Readme is non final one cause document isn't published 2. - In /manifests/04-qdrant-client we have a data.csv file - I didn't put "START/END" there - is this fine ?

**Comment by bourgeoisor** [comment]
1) that's fine we can fix later!
2) if you're not embedding it in the tutorial, then it doesn't need these tags no

**Comment by BRV158** [comment]
> 2\. if you're not embedding it in the tutorial, then it doesn't need these tags no

This is a source for config map still .csv doesn't support comments and config map is done by regular kubectl command, hope we can skip tags here

File: databases/qdrant/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Deploying a Qdrant vector database and StatefullHA operator on GKE
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/kubernetes-engine-samples&cloudshell_tutorial=cloudshell/tutorial.md&cloudshell_workspace=gke-qdrant)
+
+This example shows how to build and deploy a PostgreSQL cluster using [Kubernetes](https://kubernetes.io) and Day 1 & 2 operations such as Monitoring with [Cloud Monitoring](https://cloud.google.com/monitoring) and disaster recovery with [Backup for GKE](https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/concepts/backup-for-gke).
```
Comment (bourgeoisor) [inline databases/qdrant/README.md:5]: That doesn't seem to align with this directory, no?

**Review by bourgeoisor — APPROVED** [review]

**Pull request merged**